pallets/sto/src/lib.rs

```diff
@@ -55,49 +56,70 @@ type Timestamp<T> = timestamp::Module<T>;
 type Portfolio<T> = portfolio::Module<T>;
 type Asset<T> = asset::Module<T>;
 
-/// Details about the Fundraiser
+/// Status of a Fundraiser.
+#[derive(Clone, PartialEq, Eq, Encode, Decode, Debug, PartialOrd, Ord)]
+pub enum FundraiserStatus {
+    /// Fundraiser does not exist.
+    Unknown,
```

Comment:
Why can't we use Option instead?

Reply:
Because there's no use of an Option. The Fundraiser object is already behind an option. I added an extra type here just because we needed a default implementation for the Enum. Using an extra `option` instead of this type will only increase the cost.

Reply:
Adding an `Option<_>` layer doesn't necessarily increase cost, https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=336f7387c8cb92c53f40c161f8493e20

Reply:
That's because rust is smart enough to treat `None` as an enum variant. However, the scale codec is not as smart. It encodes `Some` values as `01 $enum_variant`. Costs 1 extra byte of storage. Quite insignificant but I literally see no advantage of using an Option here.

Reply:
Since `Unknown` is not reachable anyway, I'd apply Adam's suggestion and make `Closed` the default.

Reply:
Yep, good points.